assets/css/nav/_bottom_nav_mobile.scss

```diff
@@ -0,0 +1,59 @@
+.m-bottom-nav-mobile {
+  background-color: $color-gray-50;
+  border-top: 1px solid $color-gray-300;
+  padding: 0.5rem 0.75rem 0.5rem 0.75rem;
+}
+
+.m-bottom-nav-mobile__links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.m-bottom-nav-mobile__link,
+.m-bottom-nav-mobile__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 0.75rem;
+  border-radius: 0.375rem;
+  font-weight: 400;
```

Comment:
Is the `font-weight` necessary here? I thought there was no text in the bottom nav. Same question applies for the `active` class below.

Reply:
resolved in 9868b3c